packages/cli/src/utils/modules/constants.ts

```diff
@@ -9,18 +9,21 @@ export const defaultModuleContracts = [
     name: "KeysWithValueModule",
     abi: KeysWithValueModuleData.abi as Abi,
     bytecode: KeysWithValueModuleData.bytecode.object as Hex,
+    placeholders: [], // TODO
```

Comment:
Why is this TODO? I can see us revisiting this and not knowing what we need "to do"

Reply:
will clarify! this isn't used/needed right now but would need to be filled in if these modules ever start using public libs

Reply:
went ahead and just moved out the placeholder logic so I could reuse it here: https://github.com/latticexyz/mud/pull/1910/commits/4fea2b3f9110f8b8c201194926658934802439ad